tests/test_diaForcedSource.py

```diff
@@ -49,16 +48,19 @@ def create_test_dia_objects(n_points, wcs, startPos=100):
 
     Returns
     -------
-    test_points : `lsst.afw.table.SourceCatalog`
+    test_points : `pandas.DataFrame`
         Catalog of points to test.
     """
-    objects = afwTable.SourceCatalog(make_dia_object_schema())
+    ids = np.arange(n_points, dtype=np.int64)
+    points = [wcs.pixelToSky(startPos + src_idx, startPos + src_idx) for src_idx in range(n_points)]
```

Comment:
Maybe this form is less work?
```suggestion
    points = [wcs.pixelToSky(startPos + src_idx, startPos + src_idx) for src_idx in ids]
```